dockerfiles/reverse-proxy/templates/alb_server.conf.erb

```diff
@@ -18,8 +18,6 @@ server {
   }
   <% end %>
 
-  keepalive_timeout 5;
```

Comment:
`keepalive_timeout` in `server` block just overwrites the `http` config. we don't need to overwrite it so I just removed this line